libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -128,7 +128,7 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
     // @Param: 2_TYPE
     // @DisplayName: Second Airspeed type
     // @Description: Type of 2nd airspeed sensor
-    // @Values: 0:None,1:I2C-MS4525D0,2:Analog,3:I2C-MS5525,4:I2C-SDP3X
+    // @Values: 0:None,1:I2C-MS4525D0,2:Analog,3:I2C-MS5525,4:I2C-MS5525 (0x76),5:I2C-MS5525 (0x77),6:I2C-SDP3X
```

Comment:
do we want index 4 to say AUTO?

Reply:
I assume you mean option 3? We can definitely do that. We would want to change the doc for ARSPD_TYPE as well.